pom.xml

```diff
@@ -21,7 +21,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>11</java.version>
         <junit.version>5.7.0</junit.version>
-        <faker.version>1.0.2</faker.version>
+        <faker.version>1.12.0-SNAPSHOT</faker.version>
         <assertj.version>3.18.0</assertj.version>
```

Comment:
Sorry, I didn't see that you reused the same property in the dependency switch. It would be important to put the final release instead of the SNAPSHOT. The question would be why you needed to upgrade the dependency version. In any case, unfortunately the easy-random project is also in maintenance mode and they probably won't make a change of this size

Reply:
Maybe you need to fork and generate a custom version to use kotlin-faker